packages/fxa-admin-panel/CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+## 1.226.2
+
+### Other changes
+
+- shared,auth,admin: Refactor attached-clients so routines can be shared with admin panel (#11817)" ([4cbc3bce3](https://github.com/mozilla/fxa/commit/4cbc3bce3))
```

Comment:
Dang, our changelog didn't like the double quote and cut something off so some of these don't have the word "revert" in front of them. When you click on the commit though, you can see that it's a revert, so it's not a big deal.